manipulate_pim_data/catalog_structure/creating_a_reference_data.rst

```diff
@@ -174,13 +174,12 @@ To check if the entities are correctly set up, use the following command:
     php bin/console doctrine:mapping:info
 
 
-Overriding the ProductValue
+Configuring the ProductValue
 ---------------------------
 
 Depending on the needs, a product can be linked to several colors or just one.
 The first case will be called *simple reference data*, while the second one will be referred to as *multiple reference data*.
 
-To link a reference data to the product, the `ProductValue` object needs to be overriden.
 This task is documented here :doc:`overriding_the_orm_product_value`.
```

Comment:
Kill this one too...

Reply:
done :)

Reply:
Not done, but it's my fault. I was not clear enough.You must remove `This task is documented here :doc:`overriding_the_orm_product_value`.`